Files/UserControls/ModernNavigationToolbar.xaml

```diff
@@ -685,7 +702,7 @@
                     ItemClick="PathViewInteract_ItemClick"
                     ItemContainerStyle="{StaticResource ListViewItemContainerStyle_NoRevealBorder}"
                     ItemsSource="{x:Bind local1:App.CurrentInstance.NavigationToolbar.PathComponents, Mode=OneWay}"
-                    ScrollViewer.HorizontalScrollBarVisibility="Auto"
+                    ScrollViewer.HorizontalScrollBarVisibility="Hidden"
```

Comment:
Is there any reason you set the scroll bar to always be hidden?

Reply:
Yes, scrollbar was interfering with navigationbar interaction, it was taking almost the height of navigationbar when path is too long.